src/frontend/screens/ObservationsList/ObservationListItem.tsx

```diff
@@ -60,7 +56,7 @@ function ObservationListItemNotMemoized({
             />
           </Text>
         </View>
-        {photos.length ? (
+        {observation.attachments.length ? (
```

Comment:
So this made me realize that we only want to show this if there are photos (not audio and video). I originally changed it because it was being done in a very clunky way, and we were still passing ALL the attachments to the PhotoStack. I have now made it typesafe and it only shows up when there are photos.

Reply:
https://github.com/digidem/comapeo-mobile/pull/536/commits/014dcbf4c0342e57a401d348a7fd1b89c3ce2877